harvard-university-of-worcester.csl

```diff
@@ -261,14 +261,14 @@
       </else>
     </choose>
   </macro>
-  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+  <citation et-al-min="3" et-al-use-first="1" sort-separator="," disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=" ">
```

Comment:
@adam3smith, I'm having trouble too.

With regular apa.csl: "(Niklas, Schneider, & Heinzle, 2005, 2010)"
With added `cite-group-delimiter="(delimiter)"`: "(Niklas, Schneider, & Heinzle, 2005, 2010)" 
With added `cite-group-delimiter="(delimiter)"`, but without `collapse="year"`: "(Niklas, Schneider, & Heinzle, 2005(delimiter)Niklas, Schneider, & Heinzle, 2010)"

The spec says for `cite-group-delimiter`: "Activates cite grouping and specifies the delimiter for cites within a cite group.". I looks like it only works if collapse is not active, but that shouldn't be necessary.